.eslintrc

```diff
@@ -43,36 +50,35 @@ rules:
   prefer-spread: 2
   prefer-template: 2
 
-globals:
-  # Meteor globals
-  Meteor: false
-  DDP: false
-  Mongo: false
-  Session: false
-  Accounts: false
-  Template: false
-  Blaze: false
-  UI: false
-  Match: false
-  check: false
-  Tracker: false
-  Deps: false
-  ReactiveVar: false
-  EJSON: false
-  HTTP: false
-  Email: false
-  Assets: false
-  Handlebars: false
-  Package: false
-  App: false
-  Npm: false
-  Tinytest: false
-  Random: false
-  HTML: false
+  # eslint-plugin-meteor
+  meteor/globals: 2
+  meteor/no-zero-timeout: 2
+  meteor/no-session: 0
+  meteor/pubsub: 2
+  meteor/core: 2
+  meteor/methods: 2
+  meteor/check: 2
+  meteor/connections: 2
+  meteor/collections: 2
+  meteor/session: [2, 'no-equal']
```

Comment:
Did you ever write `Session.equal('foo', bar)` and wonder why it didn't work? Well, I certainly did.

The option `no-equal` to this rule makes the rule prevent that and tells you to use `Session.equals` instead.